internal/cmd/group_tokens.go

```diff
@@ -110,8 +111,17 @@ var groupCreateTokenCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-
-		token, err := client.Groups.Token(group.Name, expiration, flags.ReadOnly())
+		var claim *turso.PermissionsClaim
+		if len(flags.Claim()) > 0 {
```

Comment:
We should use a more specific name. This is not a generic claim, it is a list of databases that are allowed to read attached.

Reply:
⛏️: the name should be a plural as well.

Reply:
> We should use a more specific name. This is not a generic claim, it is a list of databases that are allowed to read attached.

I agree. What do you think about `--attach-claims`? (`attach-read` is not needed since attach is going to read only)

Reply:
It is better. 

I'd prefer something like `--attach` or `--attach-to`. 
Since we are in the context of a token creation command adding the `-claims` suffix doesn't add much, IMO.

But as long as it mentions attach, I'm fine! Feel free to pick the name.

Reply:
Makes sense, going with `--attach`